crates/holochain_integrity_types/src/action.rs

```diff
@@ -571,6 +571,7 @@ pub struct DeleteLink {
 /// When used in CloseChain, this contains the new DNA hash or Agent key.
 /// When used in OpenChain, this contains the previous DNA hash or Agent key.
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, SerializedBytes, Hash)]
+#[serde(tag = "type", content = "value", rename_all = "snake_case")]
 pub enum MigrationTarget {
```

Comment:
This is part of open/close chain and migration logic. It's in the HDI. If this has leaked to the app/admin interface then I think that's probably a mistake. I would choose not to modify this one